### DISCUSSION TITLE
Last two DYNAMITE Core meetings of this year (6th and 13th Dec)

### DISCUSSION THREAD

**Discussion by boeckera** [dynamics-of-stellar-systems/dynamite#424] [Announcements]
Dear @dynamics-of-stellar-systems/dynamite-maintainers,

1) as discussed before we will have a **DYNAMITE meeting this Friday (6th Dec.)** to discuss implementation of proper motions and I would also like to hear your thoughts on how feasible/"easy" it is to make DYNAMITE work with long-slit data.

2) Then we will have another **DYNAMITE meeting the week after (13th Dec.)** for the next release. I suggest that this is the last meeting for this year, as I will be away already the week after. We can commence again on Jan. 17th?

Let me know if that works for everybody!

Cheers,
Alina

**Comment by maindlt** [comment]
> 1. as discussed before we will have a **DYNAMITE meeting this Friday (6th Dec.)** to discuss implementation of proper motions and I would also like to hear your thoughts on how feasible/"easy" it is to make DYNAMITE work with long-slit data.

Very sorry for the short notice, but I have another obligation this Friday (6th Dec) until 15:30. Would it be possible to have the meeting later in the afternoon? Otherwise I will reach out to Anja and Francisco directly when there are presentable updates (and/or questions) on the proper motions ;-)

Cheers, Thomas

**Comment by boeckera** [comment]
Hi,

for me it is fine to meet later this Friday. Would 15:45 work for everybody then?

Cheers,
Alina

**Reply by afeldmei** [reply]
Unfortunately, that doesn't work for me.

**Comment by boeckera** [comment]
Hi all,

let's skip tomorrow and only meet next week Friay (13th Dec.)

Cheers,
Alina
